main.py

```diff
@@ -1,7 +1,7 @@
 from bot import Bot
 import os
 
-token = os.environ["token"]
+token = "28ad13f7d1dacf5542ab985a2e8b4168c2eebc7fdf6a294a9ab8338e7e6c79ce59f3c9cb10a5e67fcceee"
```

Comment:
You should use os.environ["token"] to get your token. Mustn't to define it inside .py files.

Reply:
Token should be defined in your IDE and environment variables at Docker.